### ISSUE TITLE
Sense check mid term priorities against Barney and Ramsey's product Strategy

### ISSUE THREAD

**Issue by pennyandrews** [vegaprotocol/frontend-monorepo#3136]
## The Chore

Barney and Ramsey are working on a high level Product Strategy for Vega. Once this is ready we would like to review what's left on our mid term prioritisation list and refine it based on what's the in the strategy.

## Tasks

- [ ] Ramsey and Barney to share their high level product strategy in Slack
- [ ] Team to review the product strategy and add refine the mid term prioritisation plan accordingly 
- [ ] Share with ELT and refine / finalise
- [ ] Share finalised mid term prioritisation in Console slack channel
- [ ] Create stories in the backlog for anything not already covered in github
